doc/CMakeLists.txt

```diff
@@ -44,7 +44,8 @@ if(DOXYGEN_FOUND)
             COMMENT "Generating API documentation with Doxygen" VERBATIM)
 
     if (TWEENY_BUILD_DOCUMENTATION)
-        install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html DESTINATION share/doc/Tweeny)
+        include(GNUInstallDirs)
```

Comment:
Top-level CMakeLists.txt already includes `GNUInstallDirs`, I wonder if it is required to have it included here aswell?

Reply:
It is not required. I include modules I use in the same file to make it more robust against changes in the top-level CMakeLists.txt. Should I remove it?

Reply:
Makes sense. I'll accept the way it is.